packages/webview_flutter/webview_flutter/lib/src/v4/src/webview_controller.dart

```diff
@@ -0,0 +1,263 @@
+  ///    These are not yet supported in iOS WkWebView. Service workers tend to
+  ///    use this cache.
+  ///	3. Application cache.
+  Future<void> clearCache() {
+    return platform.clearCache();
+  }
+
+  /// Clears the local storage used by the WebView.
+  Future<void> clearLocalStorage() {
+    return platform.clearLocalStorage();
+  }
+
+  /// Runs the given JavaScript in the context of the current page.
+  ///
+  /// The Future completes with an error if a JavaScript error occurred.
+  Future<void> runJavaScript(String javaScript) {
+    return platform.runJavaScript(javaScript);
+  }
+
+  /// Runs the given JavaScript in the context of the current page, and returns
+  /// the result.
+  ///
+  /// The Future completes with an error if a JavaScript error occurred, or if
+  /// the type the given expression evaluates to is unsupported. Unsupported
+  /// values include certain non-primitive types on iOS, as well as `undefined`
+  /// or `null` on iOS 14+.
+  Future<Object> runJavaScriptReturningResult(String javaScript) {
+    return platform.runJavaScriptReturningResult(javaScript);
+  }
+
+  /// Adds a new JavaScript channel to the set of enabled channels.
+  ///
+  /// The JavaScript code can then call `postMessage` on that object to send a
+  /// message that will be passed to [onMessageReceived].
+  ///
+  /// For example, after adding the following JavaScript channel:
+  ///
+  /// ```dart
+  /// final WebViewController controller = WebViewController();
+  /// controller.addJavaScriptChannel(
+  ///   name: 'Print',
+  ///   onMessageReceived: (JavascriptMessage message) {
+  ///     print(message.message);
+  ///   },
+  /// );
+  /// ```
+  ///
+  /// JavaScript code can call:
+  ///
+  /// ```javascript
+  /// Print.postMessage('Hello');
+  /// ```
+  ///
+  /// to asynchronously invoke the message handler which will print the message
+  /// to standard output.
+  ///
+  /// Adding a new JavaScript channel only takes affect after the next page is
+  /// loaded.
+  ///
+  /// A channel [name] cannot be the same for multiple channels.
+  Future<void> addJavaScriptChannel(
+    String name, {
+    required void Function(JavaScriptMessage) onMessageReceived,
+  }) {
+    assert(name.isNotEmpty);
+    return platform.addJavaScriptChannel(JavaScriptChannelParams(
+      name: name,
+      onMessageReceived: onMessageReceived,
+    ));
+  }
+
+  /// Removes the JavaScript channel with the matching name from the set of
+  /// enabled channels.
+  ///
+  /// This disables the channel with the matching name if it was previously
+  /// enabled through the [addJavaScriptChannel].
+  Future<void> removeJavaScriptChannel(String javaScriptChannelName) {
+    return platform.removeJavaScriptChannel(javaScriptChannelName);
+  }
+
+  /// The title of the currently loaded page.
+  Future<String?> getTitle() {
+    return platform.getTitle();
+  }
+
+  /// Set the scrolled position of this view.
```

Comment:
We should use a consistent verb form, so this should be "Sets". Most of the methods after this point need the same change.